content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,171 @@
+# CI for ML
+
+## Motivation
+
+ML engineers and data scientists face many unique challenges. Some of these
+challenges can be solved by adopting first-citizen concepts from traditional
+software engineering and development. Probably the most common adoption is
+versioning control. Some other, unfortunately less common, could be unit testing
+data processing steps of a pipeline. Here is a non exhaustive list of a few
```

Comment:
"data processing steps of a pipeline" -> "defining reproducible workflows or pipelines" ?

Reply:
I do refer here to the need of implementing unit tests aimed at testing the logic in the data processing. I'm not sure I understand @jorgeorpinel point...

Reply:
Oh, I see, I got it wrong. "unit testing data processing steps of a pipeline" is a little hard to read I guess.

Reply:
Maybe just "data processing stages"? ("stage" is a more DVC-specific term, which you can even link to https://dvc.org/doc/command-reference/run if you want.) You can probably use the key work "pipeline" in other places

Reply:
I rephrased the sentence; let me know if it is easier to understand. I didn't link to `dvc run`, trying to keep it simpler.